src/app/sessions/services/sessions-service.ts

```diff
@@ -16,6 +16,12 @@ export class SessionsService {
     constructor(private http: HttpClient, private config: AppConfig) {
     }
 
+    getSession(sessionId: string | String): Observable<any> {
+        return this.http
+            .get<Session>(`${this.config.getApiUrl()}/sessions/${sessionId}`)
+            .pipe(map(data => {console.log(normalize(data, session)); return normalize(data, session)}));
```

Comment:
console.log, Observable<any>? Coudn't be session?

Reply:
the function return type has the same problem as for the upper comment